spacy/tokens/span.pyx

```diff
@@ -656,7 +656,7 @@ cdef class Span:
         else:
             return self.doc[root]
 
-    def char_span(self, int start_idx, int end_idx, label=0, kb_id=0, vector=None, id=0, alignment_mode="strict", span_id=0):
+    def char_span(self, int start_idx, int end_idx, label=0, kb_id=0, vector=None, alignment_mode="strict", span_id=0):
```

Comment:
Could we also make everything except for `start_idx` and `end_idx` kwd-only args?

Reply:
In a separate PR, sure.